ext/spl/tests/bug75717.phpt

```diff
@@ -8,7 +8,7 @@ function flatten(array $nestedArraysAndStrings){
     $iter = new RecursiveIteratorIterator(
         new RecursiveArrayIterator($nestedArraysAndStrings));
     foreach($iter as $leaf){ $flat[] = $leaf; }
-    return join(NULL, $flat);
+    return join("", $flat);
```

Comment:
Did this result in an error previously? I wouldn't expect it to (no strict_types + internal function).

Reply:
Ah yes, it came from the last PR. I'll revert it soon.